include/taskomat/exceptions.h

```diff
@@ -22,10 +22,14 @@
 
 // SPDX-License-Identifier: LGPL-2.1-or-later
 
-#ifndef TASKOMAT_ERROR_H_
-#define TASKOMAT_ERROR_H_
+#ifndef TASKOMAT_EXCEPTIONS_H_
+#define TASKOMAT_EXCEPTIONS_H_
```

Comment:
We should soon rename all this `TASKOMAT` to `TASKOLIB` as suggested by you elsewhere (the names of our values in the Lua registry it was; while `TASK` would be too generic imho).

Reply:
Agreed, but that is for a separate MR.